New package: VirtualDrivers.Virtual-Display-Driver version 25.7.23

ye4241 opened the pull request:
### Pull request has been created with [komac](https://github.com/russellbanks/Komac) v2.12.1 :rocket:

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/280207)

ye4241 committed:
New package: VirtualDrivers.Virtual-Display-Driver version 25.7.23

ye4241 committed:
Add nested installer configuration for VirtualDrivers.Virtual-Display-Driver version 25.7.23

ye4241 committed:
Fix executable filename in nested installer configuration

- Changed 'VDD.Control.exe' to 'VDD Control.exe' to match actual filename in ZIP
- This resolves the installation validation error

stephengillie commented:
Verify:
<img width="855" height="729" alt="image" src="https://github.com/user-attachments/assets/a1222d3d-5328-45cf-9d30-0e72e99c6e59" />
<img width="444" height="148" alt="image" src="https://github.com/user-attachments/assets/07234f94-6efb-403e-a79d-23bd8c90731e" />

ye4241 commented:
@MikeTheTech or @Jocke is the manager or developer of this project? Could you assist in confirming the issue with the version number?

jocke commented:
I have nothing to do with this, please remove my username (@jocke) from the thread.

ye4241 committed:
Add DisplayVersion for VirtualDrivers.Virtual-Display-Driver 25.7.23

- Add AppsAndFeaturesEntries with DisplayVersion 25.7.26
- Resolves mismatch between package version (25.7.23) and registry display version (25.7.26)

ye4241 commented:
@stephengillie Since I couldn't reach the author of this open-source project to inquire about the specific reason, I temporarily added the `DisplayVersion` attribute for now.

stephengillie commented:
Verify:
<img width="828" height="637" alt="image" src="https://github.com/user-attachments/assets/c1b75825-637f-4d46-877d-c1c8c125d3f0" />
<img width="594" height="718" alt="image" src="https://github.com/user-attachments/assets/03391acf-4cbb-4c0c-b48e-db0c86f5a183" />
